cluster/gce/gci/configure-helper.sh

```diff
@@ -826,7 +826,8 @@ function create-master-auth {
   fi
   # (TODO/cloud-provider-gcp): Figure out how to inject automatically
   if [[ -n "${PDCSI_CONTROLLER_TOKEN:-}" ]]; then
-    append_or_replace_prefixed_line "${known_tokens_csv}" "${PDCSI_CONTROLLER_TOKEN},"        "system:pdcsi-controller,uid:system:pdcsi-controller,system:masters"
+    append_or_replace_prefixed_line "${known_tokens_csv}" "${PDCSI_CONTROLLER_TOKEN},"        "system:pdcsi-controller,uid:system:pdcsi-con
```

Comment:
This line looks strange, broken line breaking?

Reply:
Fixed.